paddle/operators/adagrad_op.h

```diff
@@ -29,25 +37,43 @@ class AdagradOpKernel : public framework::OpKernel<T> {
     param_out_tensor->mutable_data<T>(ctx.GetPlace());
     moment_out_tensor->mutable_data<T>(ctx.GetPlace());
```

Comment:
Does it make sense to move lines 26 - 30 into the if else block since they are only used for LoDTensor type?